src/ethereum/EtherscanButton/index.stories.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import styled from 'styled-components';
+
+import EtherscanButton from './index';
+import { Text } from '../..';
+
+export default {
+  title: 'Ethereum/EtherscanButton',
+  component: EtherscanButton,
+  parameters: {},
+};
+
+const StyledText = styled(Text)`
+  margin-right: 5px;
+`;
+
+export const SimpleEtherscanButton = (): React.ReactElement => (
+  <>
+    <StyledText size="md">An Address example </StyledText>
+    <br />
+    <EtherscanButton
+      type="address"
+      value="0xda6786379ff88729264d31d472fa917f5e561443"
+    />
+    <br />
+    <StyledText size="md">A Transaction example </StyledText>
```

Comment:
why does it have an empty space after "example"? since it's followed by <br /> I don't think it's needed

Reply:
done!